src/api/firebase.js

```diff
@@ -190,16 +190,25 @@ export async function updateItem(listPath, itemId, dateLastPurchased) {
 	}
 	const item = itemSnapshot.data();
 
-	const effectiveDateLastPurchased = dateLastPurchased
-		? dateLastPurchased.toDate()
-		: new Date();
-
 	const previousEstimate = item.daysUntilNextPurchase;
 
-	const daysSinceLastPurchase = getDaysBetweenDates(
-		new Date(),
-		effectiveDateLastPurchased,
-	);
+	const itemCreated = item.dateCreated;
+
+	let daysSinceLastPurchase;
+
+	if (dateLastPurchased) {
+		daysSinceLastPurchase = getDaysBetweenDates(
+			new Date(),
+			dateLastPurchased.toDate(),
+		);
+	} else if (!getDaysBetweenDates(new Date(), itemCreated.toDate())) {
```

Comment:
```suggestion
        } else if (getDaysBetweenDates(new Date(), itemCreated.toDate()) !== 0) {
```
minor suggestion to show we meant zero and not null or false